site/website_cloudfront.tf

```diff
@@ -165,6 +163,39 @@ resource "aws_cloudfront_distribution" "diagram" {
   }
 }
 
+# source of the cloudfront logs
+resource "aws_cloudwatch_log_delivery_source" "cloudfront_logs_source" {
+  region       = "us-east-1"
```

Comment:
Why  these regions all `us-east-1` and not `eu-west-2`?

Reply:
cloudfront is a global service but managed from us-east-1